doc/changes.md

```diff
@@ -1,5 +1,76 @@
 # Changelog
 
+## Release v0.3.1 (2021/03/14)
+
+[![Digital Object Identifier for PyGMT v0.3.1](https://zenodo.org/badge/DOI/10.5281/zenodo.3781524.svg)](https://doi.org/10.5281/zenodo.3781524)
+
+### Highlights
+
+*🎉 **Multiple bug fixes and a reorganized gallery** 🎉
+*
+*
```

Comment:
Still need a couple of highlight items here team! What are the key changes to the documentation we want to highlight to our users? Do we want to mention that PyGMT v0.3.1 is the last release to support GMT 6.1.1 (and that the next version will require GMT 6.2.0)?

Reply:
For the documentation, I'd like to recommend [Cartesian, circular, and geographic vectors](https://www.pygmt.org/dev/gallery/lines/vectors.html#sphx-glr-gallery-lines-vectors-py)

> Do we want to mention that PyGMT v0.3.1 is the last release to support GMT 6.1.1 (and that the next version will require GMT 6.2.0)?

Sounds good.

Reply:
Works for me! Updating now.